src/AppBundle/Mailjet/Message/EventRegistrationConfirmationMessage.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace AppBundle\Mailjet\Message;
+
+use AppBundle\Entity\EventRegistration;
+use Ramsey\Uuid\Uuid;
+
+final class EventRegistrationConfirmationMessage extends MailjetMessage
```

Comment:
There are unit tests for this kind of class (in Mailjet/Message). Can you add one ?